pkg/consolegraphql/accesscontroller/access_controller.go

```diff
@@ -91,6 +91,7 @@ func (k *kubernetesRbac) CanRead(obj interface{}) (bool, error) {
 			Resource:  plural.Resource, // Resource is Kind
 			Version:   gvk.Version,
 			Namespace: o.Name,
+			Name:      o.Name,
```

Comment:
when I wrote access_controller.go, i was guided by what `oc auth can-i get namespaces --loglevel=9` did.  It passes only the `namespace` field and not the `name`.

Reply:
`oc auth can-i get namespaces --loglevel=9` checks if you can 'get' namespaces in general, but not a specific namespace, which is the permission I was using.

If you create a role using resources: ["namespaces"] and resourceNames: ["foo"] to grant permissions to a single namespace, then omitting the 'Name' in the ssar will cause it to fail.

Reply:
so that would sound like an existing defect against 0.31?  however, I know we test case where the user has access to a only a single namespace and the namespace filtering works as expected.

Reply:
I'll check if this is only a problem for Kubernetes or not. I think that if the namespace is created by the user, then you automatically get some permissions applied. In my case, the namespace is created by someone else, and the user has no permissions at all to begin with.